api/transactions.js

```diff
@@ -230,7 +231,12 @@ function setTransactionBitFlags(transaction, options) {
  * @param {Error} error
  * @param {Transaction} transaction
  */
-function getTransaction(api, account, identifier, callback) {
+function getTransaction(api, account, identifier, ledger, callback) {
+  if (typeof ledger === 'function') {
```

Comment:
I should have said "signature" instead of "prototype". Sometimes they are used interchangeably, but I think signature is more clear.